modules/core/src/main/java/org/apache/ignite/internal/IgniteKernal.java

```diff
@@ -355,7 +355,7 @@
      * <p>
      * Value is {@code 30 sec}.
      */
-    private static final long PERIODIC_STARVATION_CHECK_FREQ = 1000 * 30;
+    public static final long PERIODIC_STARVATION_CHECK_FREQ = 1000 * 30;
```

Comment:
I think it's safe to rename constant to `DFLT_PERIODIC_STARVATION_CHECK_FREQ` like the other constants named.